### PR TITLE
fix: derive the branch from version tag

### PR DESCRIPTION
**Pull request by HofmannZ**
This PR fixes the automatic release

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -155,21 +155,25 @@ jobs:
...
 
             # patch version
-            ~/yq w pubspec.yaml version $CIRCLE_TAG > patched_pubspec.yaml
+            ~/yq w pubspec.yaml version ${CIRCLE_TAG%"@beta"} > patched_pubspec.yaml
             rm pubspec.yaml
             mv patched_pubspec.yaml pubspec.yaml
 
             # patch graphql dependency version
-            ~/yq w pubspec.yaml dependencies.graphql ^$CIRCLE_TAG > patched_pubspec.yaml
+            ~/yq w pubspec.yaml dependencies.graphql ^${CIRCLE_TAG%"@beta"} > patched_pubspec.yaml
```
Comment (micimize) [inline .circleci/config.yml]: will we need to change this when merging `beta` into `master`?

Comment (HofmannZ) [reply .circleci/config.yml]: Nope! It's bash for: `optionally strip "@beta"`.

**Pull request merged**